deployment/ansible/roles/model-my-watershed.geoprocessing/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Update CA Certificates
+  shell: "update-ca-certificates -f"
```

Comment:
We have to do this for AWS credentials to work with OpenJDK. Otherwise we get this error:

```
Unable to execute HTTP request: java.lang.RuntimeException: Unexpected error: java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty
```

Does this make sense, or is it something completely out of left field? @hectcastro @tnation14 

I got this from https://stackoverflow.com/a/29313285/6995854